### ISSUE TITLE
Placing Windows form Macaw's Windows makes a water source

### ISSUE THREAD

**Issue by Coldphoenix94** [Direwolf20-MC/BuildingGadgets#579]
**IMPORTANT:** Before continuing, please ensure that Building Gadgets (and any other mods involved) is updated to the latest available version
----------------------------------------------------------------------------
After deleting this template, please provide the following information:
* Forge version 35.1.15
* Building Gadgets version 3.7.3
* *If the issue is a crash*; An external link to the crash report
* How to reproduce the issue Placing a Diorite Tall Windows with the Building Gadjet
* A screenshot, if applicable https://imgur.com/a/zib20Ek

It makes a water source insite the window. Breaking the windows does not remove the water source. Kind of weird one

**Comment by MichaelHillcox** [comment]
@hvdklauw this one might be for you

**Comment by hvdklauw** [comment]
Haha, yeah sounds familiar, let's have a look at their default state

**Comment by hvdklauw** [comment]
Hmm can't find the source :(

**Comment by Hanawa02** [comment]
The same happens for Illuminant Panel, Illuminant Fixture and Illuminant Slab from the Simply light mod.

**Comment by hvdklauw** [comment]
Ok, maybe we should just always force waterlogged to false :S

**Comment by MichaelHillcox** [comment]
So many mods do this wrong apparently :P

**Comment by Staegrin** [comment]
Also caused when pasting Dark Shingle Roof from engineer's decor. 
Only the normal variant and not the one with a chimney or window in it.
Minecraft: 1.16.5
Forge: 36.0.1
engineer's decor version: engineersdecor-1.16.4-1.1.7
building gadgets vesion: buildinggadgets-1.16.4-3.7.3

**Comment by hvdklauw** [comment]
Tomorrow I'll just force the waterlogged to false on all the blocks, water is cheap unless you have a really weird modpack in which case we might not want to duplicate it anyway.

**Comment by b52src** [comment]
Same issue happens with engineer's décor windows as well

**Comment by MichaelHillcox** [comment]
Unfortunately not, it's on our radar to fix though :D

**Comment by b52src** [comment]
Curious if i take the debug stick and set all the blocks waterlog state to false manually before I copy the building is a valid workaround, not sure if anyone has tried it.

**Comment by hvdklauw** [comment]
Yes, we copy certain states from the blocks to copy and reset others to their default values. Weirdly enough some blocks have a default state that makes them waterlogged while when the player places them they are not.

So yes, it's kinda buildinggadget doing it, but also not. The default state of a block should never be waterlogged.

**Comment by b52src** [comment]
ok got it 
@stfwi
stfwi/engineers-decor/issues/162 created for this on Engineers Decor repo

**Comment by stfwi** [comment]
Hi, gonna take a look at/fix at the weekend in the ED mod. I agree the default states should be defined correctly in the mods providing the blocks, workarounds here may have other negative side effects. Cheers!

**Comment by Pentasis** [comment]
Just noticed this when trying to exchange some blocks for windows from Wile's Collection in the 1.18.1 version. I'll also report it to them
![Screenshot (65)](https://user-images.githubusercontent.com/7046024/150122865-3f6cc6b1-c949-43c3-a396-8e9cb87e00f6.png)
.
